src/components/TextWidth.vue

```diff
@@ -22,17 +22,17 @@
 </script>
 
 <style lang="scss" scoped>
-@import "@/styles/_variables.scss";
+  @import '@/styles/_variables.scss';
```

Comment:
`yarn lint` does do this indentation correctly, I'm not sure why it wasn't there in this case before this diff, maybe it just hadn't been invoked for a while on this repo since Michael made his updates to the tooling.